data_linked_in_ssr_manual/loudspeakers_1-2.asd

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<asdf>
+  <header>
+    <name>.</name>
+    <description>
+      See https://ssr.readthedocs.io/en/latest/use-cases.html for an explanation of what this file does.
```

Comment:
I prefer the link https://ssr.readthedocs.io/use-cases.html, but this will only work after the next SSR release.

Reply:
Updated